### DISCUSSION TITLE
Downloading update in incognito mode

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#1780] [Q&A]
I am on the last version before this update. I am trying to update. But the download starts in incognito mode. Why?

The after some times I saw a notification from bromite saying authorization is needed, I think it is for authorization and installing application on Android.

Since it was dl'ed on incognito, the history isn't available.

I then went to look for the file in a file manager. Found nothing!

Don't just download the APK in incognito. Don't try to automatically install the download.

**Comment by uazo** [comment]
> But the download starts in incognito mode. Why?

simply because the download is not announced, in this way nothing is logged. consider that the files reside on github and although to date it does not send any cookies when we open the link, but we have no guarantee that it will continue to do so.

> The after some times I saw a notification from bromite saying authorization is needed, I think it is for authorization and installing application on Android.

no, there is no automatic installation.

> I then went to look for the file in a file manager. Found nothing!

check where you have the download folder in the settings, you are probably pointing to a folder not accessible by the file manager (are you on A11?)

**Reply by ghost** [reply]
> are you on A11?

No, I am on Android 10 Go.
